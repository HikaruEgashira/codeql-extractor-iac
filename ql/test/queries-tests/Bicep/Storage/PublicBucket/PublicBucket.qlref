security/Bicep/Storage/PublicAccess.ql