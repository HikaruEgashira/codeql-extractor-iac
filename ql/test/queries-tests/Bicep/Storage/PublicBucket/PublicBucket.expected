| storage.bicep:11:1:16:1 | ResourceDeclaration | Public Blob Container resource. |
